### PR TITLE
Fix some compiler warnings

### PR DESCRIPTION
**Pull request by cprogrammer1994**
- sinf to btSin
- acosf to btAcos
- float to btScalar
- 1e9 to ... (1e9 is treated as double)

### CODE REVIEW

File: src/LinearMath/btQuickprof.cpp [hunk 1]
```diff
@@ -218,7 +218,7 @@ unsigned long long int btClock::getTimeNanoseconds()
 		QueryPerformanceCounter(&currentTime);
 		elapsedTime.QuadPart = currentTime.QuadPart - 
 			m_data->mStartTime.QuadPart;
-		elapsedTime.QuadPart *= 1e9;
```
Comment (cprogrammer1994) [inline src/LinearMath/btQuickprof.cpp]: `1e9` is treated as `double`
`elapsedTime.QuadPart` is not a `double`

File: src/BulletDynamics/Featherstone/btMultiBody.h [hunk 2]
```diff
@@ -616,7 +616,7 @@ void addJointTorque(int i, btScalar Q);
     void operator=(const btMultiBody &);  // not implemented
 
 
-	void solveImatrix(const btVector3& rhs_top, const btVector3& rhs_bot, float result[6]) const;
```
Comment (cprogrammer1994) [inline src/BulletDynamics/Featherstone/btMultiBody.h]: this function is not used.

Comment (erwincoumans) [inline src/BulletDynamics/Featherstone/btMultiBody.h]: it is a left-over from older code before we introduced spatial algebra. We can remove it.

**Comment by erwincoumans** [comment]
thanks for the improvements!

**Pull request merged**